package.json

```diff
@@ -66,7 +66,7 @@
     "new-version-pr-generator": "node ./scripts/new-version-pr-generator",
     "copy-polaris-tokens": "rimraf ./src/styles/polaris-tokens && shx cp -r ./node_modules/@shopify/polaris-tokens/dist ./src/styles/polaris-tokens",
     "prepublish": "in-publish && yarn run build || :",
-    "postpublish": "in-publish && npm-run-all new-version-pr-generator",
+    "postpublish": "in-publish",
```

Comment:
[in-publish](https://www.npmjs.com/package/in-publish) just acts as a gateway so this command doesn't run when you run `yarn install` (yes you read that right. see https://github.com/npm/npm/issues/10074).

So now there's no script too run, we can get rid of this too.

```suggestion
```